tools/edrumulus_gui.py

```diff
@@ -50,12 +50,14 @@
 # tables
 pad_types_dict = {"PDA120LS Roland Mesh Pad":18, "PDX100 Roland Mesh Pad":19, "PD120 Roland Mesh Pad":0, \
                     "PD85 Roland Mesh Pad":1, "PDX8 Roland Mesh Pad":15, "DIABOLO12 drum-tec Mesh Pad":10, \
+                    "MPS-750X Millenium Mesh Tom":21, "MPS-750X Millenium Mesh Snare":22, \
                   "PD8 Roland Rubber Pad":2, "PD6 Roland Rubber Pad":13, "PD5 Roland Rubber Pad":17, \
                     "HD1TOM Roland Rubber Pad":12, "TP80 Yamaha Rubber Pad":7, \
                   "CY8 Roland Cymbal":9, "CY6 Roland Cymbal":8, "CY5 Roland Cymbal":11, "VH12 Roland Cymbal":4, \
+                    "MPS-750X Millenium Ride":24, "MPS-750X Millenium Crash":25, "LEHHS12C Lemon Hi-Hat Cymbal":26, \
                   "KD120 Roland Mesh Kick Pad":16, "KD8 Roland Kick Pad":14, \
-                    "KD7 Roland Kick Pad":6, "KT10 Roland Kick Pedal":20, \
-                  "FD8 Roland Hi-Hat Pedal":3, "VH12CTRL Roland Hi-Hat Pedal":5}
+                    "KD7 Roland Kick Pad":6, "KT10 Roland Kick Pedal":20, "MPS-750X Millenium Kick Pad":23, \
+                  "FD8 Roland Hi-Hat Pedal":3, "VH12CTRL Roland Hi-Hat Pedal":5, "LEHHS12C Lemon Hi-Hat Pedal":27}
```

Comment:
In the Edrumulus LCD display, only the first word is shown since we do not have enough space on the display. Then, we cannot distinguish the Cymbol from the Pedal. I would recommend to use LEHHS12CCTRL as I did with the VH-12.